packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -1,38 +1,98 @@
 import * as React from 'react';
-import { resolveShorthand } from '@fluentui/react-utilities';
-import { AccordionProps, AccordionSlots, AccordionState } from './Accordion.types';
-import { useCreateAccordionContextValue } from './useAccordionContext';
+import { useControllableValue, useDescendantsInit, useEventCallback } from '@fluentui/react-utilities';
+import { AccordionDescendant, AccordionIndex, AccordionProps, AccordionSlots, AccordionState } from './Accordion.types';
 
 /**
  * Const listing which props are shorthand props.
  */
 export const accordionShorthandProps: Array<keyof AccordionSlots> = [];
```

Comment:
Are these shorthandProp array still supposed to be used even if there are no slots ?

Reply:
That's a good question. WDYT?! I can remove, don't know if we have any guidance on that 🤷🏼

Reply:
I don't see a reason for keeping it, it's dead code